.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*]
```

Comment:
Interesting. I did not know https://editorconfig.org/
It's nice to have something that doesn't require a additional node package.
But we can't assume the dev is using a supported IDE (maybe some people use nano🤣?).
More importantly if you rely only on the IDE + `.editorconfig`, you won't be able to perform a formatting check in a CI workflow.
So please remove this file and use `prettier` (`@zkitter/prettier-config`) instead

Reply:
It's also super useful to set default line endings for example, and you can check that with CI afterwards if you wish to, but sure, I'll remove it 🙂